addons/setVehicleAmmo/functions/fnc_attributeLoad.sqf

```diff
@@ -0,0 +1,220 @@
+#include "script_component.hpp"
+
+params ["_control", "_loadoutValue"];
+TRACE_2("params",_control ,_loadoutValue);
+
+GVAR(turretMagsArray) = [];
+GVAR(defaultLoad) = [];
+GVAR(deltaLoad) = [];
+GVAR(selectedTurretPath) = [];
+GVAR(selectedTreePath) = [];
+GVAR(selectedMagazine) = "";
+
+
+//Parse string into a delta loadout array:
+if (_loadoutValue isEqualTo true) then {_loadoutValue = ""}; //why?
+if (_loadoutValue != "") then {
+    _loadoutValue = call compile _loadoutValue;
+    if (_loadoutValue isEqualType []) then {
+        GVAR(deltaLoad) = _loadoutValue;
+    };
+};
+
+private _entity = (get3DENSelected "object") select 0;
+TRACE_2("",_entity,typeOf _entity);
+
+private _enableLBChangedEH = {
+    params ["_control", "_selection"];
+    TRACE_2("lbChanged eh",_control,_selection);
+    if (_selection == 0) then {
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 101) ctrlEnable false;
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 101) ctrlSetFade 0.5;
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 101) ctrlCommit 0;
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 102) ctrlEnable false;
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 102) ctrlSetFade 0.5;
+        ((ctrlParentControlsGroup _control) controlsGroupCtrl 102) ctrlCommit 0;
```

Comment:
Add ((ctrlParentControlsGroup _control) controlsGroupCtrl 101) and ((ctrlParentControlsGroup _control) controlsGroupCtrl 102) to script macros?